tamr_unify_client/models/dataset_profile.py

```diff
@@ -64,14 +65,24 @@ def attribute_profiles(self) -> list:
         """
         return self._data.get("attributeProfiles")
 
+    def refresh(self, **options):
+        """Brings this profile up-to-date if needed.
```

Comment:
Brings the profile up-to-date on the server; user will need to re-fetch the profile to get the up-to-date one.